src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Map;
+import java.util.Date;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+
+public class Event {
+
+  private long id; // Represents ID of Event; Type is long because Datastore's Key's getId() return type is long.
+  private String title; // Represents the title of the Event.
+  private ArrayList<Long> ownerOrgIds; // Holds IDs (type long) of Organizations that own the Event.
+  private ArrayList<EmbeddedEntity> partnerIdsOrNames; // Holds EmbeddedEntity that represents an Organization's ID OR name. A partnering organization could have missing ID.
+  private String description; // Represents description of the Event. e.g. "In front of church ABC"
+  private Map<Date, EmbeddedEntity> dateAndHours; // Holds dates of Event and their corresponding time(s). 
+  private String contactEmail; // Represents a single contact email for this Event.
```

Comment:
I wonder if there's a more generic Contact structure that can be used here and in the organizations as well. I can imagine contacts may also need names / terms of address?

Reply:
I will discuss this with the pod and update based on the discussion!

Reply:
After the pod discussion, we've agreed to keep it as it is because we don't feel the need to store address with the contact information.

I think that creating a contact class would be an option, too.

Reply:
When I said terms of address I meant more the "I want to be called bob".

Reply:
Sorry! That makes a lot more sense. 

Would it be better to create a contact class having attributes for 
1) first name 
2) last name
3) phone number
4) email address,

or to have separate fields within the Event class holding this information?

Reply:
The former (and you may want to do this for Organization contacts, if and when you have that as well)

Reply:
Okay, sounds good! Thank you!

I will make an issue to refactor Organization class since this pull request focuses on Event class! :)